x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -34,6 +34,7 @@ export interface Cluster {
   meta: {
     clusterId: string;
     benchmarkName: string;
+    benchmarkId: string;
```

Comment:
we need `benchmarkId` to map to the right icon

Reply:
benchmark name used to change, the name it self does not generate from Kibana so i wouldn't like to depend on that. i think using the id is good